lib/components/molecules/AtomicCarousel/AtomicCarousel.js

```diff
@@ -0,0 +1,341 @@
+      <div className="atomic-dots">
+        <ListVanilla>{getSliderDots(currentItem, setItem)}</ListVanilla>
+      </div>
+    );
+  };
+
+  /**
+   * Wrapper of slide.
+   *
+   * @param {Object} slide
+   *   the slide content that need to be wrapped.
+   */
+  const slideWrapper = (slide, id = 1) => (
+    <div key={id} className="slide-item">
+      {slide}
+    </div>
+  );
+
+  /**
+   * Show required number of items on single slide.
+   *
+   * @param {Array} content
+   *   Its the children data.
+   * @param {Number} perFrame
+   *   No. of item needs to show in single slide.
+   * @param {Number} currentScreen
+   *   Its the current slide number.
+   *
+   * @return Array
+   *   Number of slides per frame.
+   */
+  const getPerFrameItems = (content, perFrame, currentScreen) => {
+    const itemsArray = [];
+    const start = currentScreen === 1 ? 0 : perFrame * (currentScreen - 1);
+
+    if (perSlideFrame > 1) {
+      for (let i = start; i <= currentScreen * perFrame - 1; i += 1) {
+        itemsArray.push(slideWrapper(content[i], i));
+      }
+    } else {
+      itemsArray.push(slideWrapper(content[currentScreen - 1]));
+    }
+
+    return itemsArray;
+  };
+
+  /**
+   * Handling next key event.
+   *
+   * @param {Object} event
+   *   keyup Object
+   */
+  const handleNextKey = event => {
+    if (event.key === 'ArrowRight') {
+      handleNext();
+    }
+  };
+
+  /**
+   * Handling Mouse Enter event.
+   *
+   * @param {Object} event
+   *   keyup Object
+   */
+  const handleMouseEnter = () => {
+    if (mouseStatus) {
+      setAnimation(false);
+    }
+  };
+
+  /**
+   * Handling Mouse Leave event.
+   *
+   * @param {Object} event
+   *   keyup Object
+   */
+  const handleMouseLeave = () => {
+    if (mouseStatus) {
+      setAnimation(true);
+    }
+  };
+
+  /**
+   * Slider Section.
+   */
+  const getSliderSection = () => {
+    return (
+      <div className="carousel-container">
+        {data.length !== 0 && (
+          <>
+            {navigation && (
+              <div className="action previous">
+                <Button
+                  data-action="prev"
+                  className={classnames({
+                    'prev-icon': prevIcon,
+                  })}
+                  onClick={handlePrev}
+                  disabled={currentSlide === 1}
+                  onKeyUp={handlePreviousKey}
+                  title={prevLabel}
+                  type="button"
+                >
+                  <Icon path={prevIcon} alt="previous" title="previous" />
+                </Button>
+              </div>
+            )}
+            <div
+              className={classnames('carousel-slides', {
+                fadeIn: fade && isAnimationRunning,
+              })}
+              onMouseEnter={handleMouseEnter}
+              onMouseLeave={handleMouseLeave}
+            >
+              {getPerFrameItems(data, perSlideFrame, currentSlide)}
+            </div>
+            {navigation && (
```

Comment:
Why are you placing both buttons in left or right of content, they are always going to be positioned by CSS on top of content. Please being them in a single container and position the button with CSS 

This will help avoid duplicate conditions